CMakeLists.txt

```diff
@@ -34,12 +34,27 @@ add_subdirectory(src)
 add_subdirectory(tools)
 add_subdirectory(qt)
 
-# Dbus file
-install(FILES data/hfd-service.conf 
+FILE(GLOB upstart_file "${CMAKE_CURRENT_SOURCE_DIR}/data/hfd-service.conf")
+FILE(GLOB systemd_file "${CMAKE_CURRENT_SOURCE_DIR}/data/hfd-service.service")
+FILE(GLOB dbus_file "${CMAKE_CURRENT_SOURCE_DIR}/data/com.lomiri.hfd.conf")
+
+# Upstart file
+install(FILES ${upstart_file}
```

Comment:
Maybe add an option to not install the upstart file at all? I don't think there are many people that have a need for it